docs/reference/mcp/tool-schema.md

```
@@ -15,6 +15,22 @@ Code references:
 - `src/mcp/schema.rs`
 - `src/mcp/server.rs`
 
+## Task-Augmented Calls
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)